src/auth/jwt.rs

```diff
@@ -0,0 +1,72 @@
+pub use biscuit::jwa::SignatureAlgorithm;
+
+use super::{ActixResult, ErrorUnauthorized};
+use crate::info;
+use biscuit::{jws::Secret, Empty, Validation, ValidationOptions, JWT};
+
+#[derive(Clone, Default)]
+pub struct ClaimCode {
+    pub nbf: bool,
+    pub exp: bool,
+}
+
+impl ClaimCode {
+    pub fn disable_all() -> Self {
+        Self::default()
+    }
+
+    pub(crate) fn validate(&self, secret: &[u8], token: &str, algo: SignatureAlgorithm) -> ActixResult<()> {
+        use SignatureAlgorithm::*;
+        let token = JWT::<Empty, Empty>::new_encoded(token);
+        let secret = match algo {
+            RS256 | RS384 | RS512 | ES256 | ES384 | ES512 | PS256 | PS384 | PS512 => Secret::PublicKey(secret.to_vec()),
+            HS256 | HS384 | HS512 => Secret::Bytes(secret.to_vec()),
```

Comment:
why are you matching all these algos? there is only one algo standard that we are using. HMAC512

Reply:
I though this is a library crates 🤔
cc: @Ujang360

Reply:
per algo?

Reply:
algo is an enum config ([SignatureAlgorithm](https://docs.rs/biscuit/0.4.1/biscuit/jwa/enum.SignatureAlgorithm.html)). Hmm, I think I should rename `algo` with `config_algo` or `select_algo` to make it clear.

Reply:
This is debatable, even more the design of this library/crate hasn't been decided yet. But we have the tendency to implement only what we need for Bitwyre Repos.

So, I think HMAC512 only for our `Websocket Core`

Reply:
Wait a minute, HS512 or RS256?

Reply:
LoL right! For JWT it is `RS256`